src/components/QuizComponent.re

```diff
@@ -142,17 +132,40 @@ let showSuccess = quiz => {
   </div>;
 };
 
-let showQuiz = (quiz, questions, setState, state) => {
+module Link = {
```

Comment:
A `Link` component has been added that handles click events specially - pushing the href to history

Reply:
Should be make it `history.replace`?